test/handlebars-layouts.spec.js

```diff
@@ -1,36 +1,31 @@
 /*eslint-env mocha */
 'use strict';
 
-var handlebarsLayouts = require('../index'),
-	expect = require('expect');
+var expect = require('expect');
 
 describe('handlebars-layouts spec', function () {
-	var count, hbs;
+	var count, handlebars, handlebarsLayouts;
 
 	beforeEach(function () {
 		count = 0;
 
-		hbs = {
-			partials: {
-				foo: function (data) {
-					count++;
-
-					return data && data.foo || '';
-				}
-			},
-			registerHelper: function (helpers) {
-				count++;
-
-				expect(helpers.extend).toBeA(Function);
-				expect(helpers.embed).toBeA(Function);
-				expect(helpers.block).toBeA(Function);
-				expect(helpers.content).toBeA(Function);
-			}
-		};
+		// Delete
+		delete require.cache[require.resolve('handlebars')];
+		delete require.cache[require.resolve('../index')];
+
+		// Reload
+		handlebars = require('handlebars');
+		handlebarsLayouts = require('../index');
```

Comment:
This is more or less a copy pasta for the e2e.js test file, don't know if we must / can dedupe this.